app/views/dams_objects/embed.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+   prefix = (defined?(componentIndex)) ? "component_#{componentIndex}_" : ''
+   objid = @document['id']
+   cmpid = (defined?(componentIndex)) ? "#{componentIndex}" : '0'
```

Comment:
Same as above.

Reply:
I updated this code to use params['cmp'] as suggested. Fixed bug.